octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
+        int c = a + b;
+
+        System.out.println("Sum of a and b is: " + c);
+
```

Comment:
[Severity: Minor] Consider adding a newline at the end of the file.
This is a common practice in many programming languages and helps with version control diffs and readability in some editors.
Rationale: Ending files with a newline is a convention that can prevent issues in certain tools and environments.
Reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline